infrastructure/kctf/scripts/setup/gcsfuse.sh

```diff
@@ -0,0 +1,39 @@
+#!/bin/bash
```

Comment:
when do I need to run this script? Is this automatic?

Reply:
Probably cluster/create.sh maybe? I have it separate for now, as it's a bit more intrusive (changes IAM policies, creates a bucket, requires gsutil), but it probably makes sense to mix it.